### DISCUSSION TITLE
How can I cancel free github hosting for my custom domain

### DISCUSSION THREAD

**Discussion by coderMojnu** [community/community#23218] [Other Feature Feedback, Questions, & Ideas]
<p>I use GitHub Student Developer Pack offer to take a .me domain on namecheap, when I order <a href="http://honestschool.me" rel="noopener nofollow ugc">honestschool.me</a> domain I choose 1 year free GitHub hosting service. That’s the problem. I want to remove GitHub hosting for my domain name, and I want to add another hosting provider service. But I couldn’t do that. Please help me.</p>
<p>When I visit <a href="http://honest.me" rel="noopener nofollow ugc">honest.me</a> website. GitHub show me a page (screenshot below).</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/4/d4ae7f8b01126a0aa70bdecbf8521dbb1e4aa90d.jpeg" data-download-href="/uploads/short-url/ult5ezuVrIJSieMMKUEd4l4VTEF.jpeg?dl=1" title="github-error" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/61357402/181075638-3d911b7d-594b-4839-bb57-80e529596832.jpeg" alt="github-error" data-base62-sha1="ult5ezuVrIJSieMMKUEd4l4VTEF" width="690" height="304" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/4/d4ae7f8b01126a0aa70bdecbf8521dbb1e4aa90d_2_690x304.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/d/4/d4ae7f8b01126a0aa70bdecbf8521dbb1e4aa90d_2_1035x456.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/d/4/d4ae7f8b01126a0aa70bdecbf8521dbb1e4aa90d.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/4/d4ae7f8b01126a0aa70bdecbf8521dbb1e4aa90d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">github-error</span><span class="informations">1342×592 19.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/codermojnu">@coderMojnu</a>, welcome to the GitHub Support Community! If you want to point your domain towards a different service then you’ll need to log into your Namecheap account and change the DNS records to point towards your new host. There’s instructions on how to do this here:</p>
<p><a href="https://www.namecheap.com/support/knowledgebase/article.aspx/767/10/how-to-change-dns-for-a-domain/" class="onebox" target="_blank" rel="noopener">https://www.namecheap.com/support/knowledgebase/article.aspx/767/10/how-to-change-dns-for-a-domain/</a></p>
<p>You’ll need to get the new DNS settings from your new hosting service, they should tell you what records you need to add.</p>

**Comment by coderMojnu** [comment]
<p>thank you very much <a class="mention" href="/thomasshaped">@thomasshaped</a>, sir. great answer.  :heart:</p>

**Comment by de-marauder** [comment]
Hello, please I am a similar issue to this. I obtained a de-marauder.me domain from namecheap using github's student pack now all my repos with github pages resolve to de-marauder.me when I go to de-marauder.github.io. 

I want to change this behaviour and have de-marauder.github.io as the default again. I have read most of the recommendations on here and I have tried deleting all DNS records in the namecheap account associated with this domain (de-marauder.me). I did this over 2 weeks ago so the DNS should have propagated by now but the problem still persists. 

I also tried changing my username while troubleshooting and that seemed to fix the issue for the new username but I had to switch back as changing my username is not an option right now. When I changed username, de-marauder.github.io was still resolving to de-marauder.me

How can I fix this please?

![image](https://github.com/community/community/assets/65220956/a8dd7276-14eb-4216-87d6-765a2b47db08)
The image shows that github pages is using a custom domain at the top. However, There is no entry at the bottom where it say custom domains. Please any help with this is appreciated

**Reply by gensahilsingh** [reply]
Same problem with me, I want to host again on github.io but I can't remove the namecheap domain.

**Reply by codedbyMojnu** [reply]
Delete CNAME file from your repository and Try Please.

**Reply by dholandac** [reply]
I have the same problem and there's no CNAME file in my repo.

**Comment by sachinbhandari691** [comment]
Please help... My domain is use by GitHub side for what I don't remember 😇

**Comment by AJ-Walker** [comment]
# Fixing "domain.me" Domain Issue in GitHub Pages

If you're still facing the issue of not being able to remove the **`domain.me`** domain from GitHub Pages, you can follow these steps.  

There are two ways to solve this:

---

## Method 1: Remove CNAME from `username.github.io` Repository

1. Go to your **GitHub profile**.  
2. Click on the **Repositories** tab.  
3. Search for the repository named **`your-username.github.io`**.  
4. Open the repository.  
5. Locate and **remove the `CNAME` file**.  

✅ This should remove the **`domain.me`** domain from GitHub Pages.

---

## Method 2: Search for CNAME Globally

1. Use the **global search bar** in GitHub and type **`CNAME`**, then press **Enter**.  
2. Look through the commits and check for entries containing **`domain.me`**.  
3. Remove or update the commit(s) where the domain was set.  

---

After completing either method, your GitHub Pages should no longer point to **`domain.me`**.
Hope this helps anyone still stuck with the issue!
